include/ppx/application.h

```diff
@@ -427,6 +440,8 @@ class Application
 #if defined(PPX_BUILD_XR)
     XrComponent mXrComponent;
     uint32_t    mDebugCaptureSwapchainIndex = 0;
+    uint32_t    mUISwapchainIndex           = 0;
+    uint32_t    mStereoScopicSwapchainIndex = 0;
```

Comment:
nit: stereoscopic is a single word, so probably should be  `mStereoscopicSwapchainIndex`

Reply:
fixed